clang/docs/LanguageExtensions.rst

```diff
@@ -706,6 +706,12 @@ Unless specified otherwise operation(±0) = ±0 and operation(±infinity) = ±in
                                              representable values for the signed/unsigned integer type.
  T __builtin_elementwise_sub_sat(T x, T y)   return the difference of x and y, clamped to the range of        integer types
                                              representable values for the signed/unsigned integer type.
+ T __builtin_elementwise_maximum(T x, T y)   return x or y, whichever is larger. If exactly one argument is   integer and floating point types
+                                             a NaN, return the other argument. If both arguments are NaNs,
```

Comment:
This doesn't fully explain the semantics, and I'd like to avoid trying to re-explain all the details in every instance of this. Can you just point this to some other description of the semantics?

Reply:
Added link to LangRef table.